### PR TITLE
FIX - php V8 lists $totalarray['val']

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# Instructions
*This is a template to help you make good pull requests. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *only keep the "FIX", "CLOSE" or "NEW" section* (use uppercase to have the PR appears into the ChangeLog, lowercase will not appears)
- *follow the project [contributing guidelines](/.github/CONTRIBUTING.md)*
- *replace the bracket enclosed texts with meaningful information*

# FIX|Fix #[*issue_number Short description*]
[*Long description*]

# CLOSE|Close #[*issue_number Short description*]
[*Long description*]

# NEW|New [*Short description*]
[*Long description*]

### CODE REVIEW

File: htdocs/comm/propal/list.php [hunk 1]
```diff
@@ -1845,7 +1845,7 @@
 			if (!$i) {
 				$totalarray['pos'][$totalarray['nbfield']] = 'p.total_ht';
 			}
-			$totalarray['val']['p.total_ht'] += $obj->total_ht;
+			isset($totalarray['val']['p.total_ht'])?$totalarray['val']['p.total_ht'] += $obj->total_ht:$totalarray['val']['p.total_ht'] = $obj->total_ht;
```
Comment (frederic34) [inline htdocs/comm/propal/list.php:1848]: image
$totalarray['val']['p.total_ht'] is set here, it's not enough ?

Comment (bb2a) [reply htdocs/comm/propal/list.php:1848]: in my instance i have a php warning
image

Comment (frederic34) [reply htdocs/comm/propal/list.php:1848]: maybe an external module resetting totalarray in hook printFieldListTitle ?

Comment (bb2a) [reply htdocs/comm/propal/list.php:1848]: in doubt i have comment lines 
`$reshook = $hookmanager->executeHooks('printFieldListTitle', $parameters); // Note that $action and $object may have been modified by hook
	print $hookmanager->resPrint;`

the result is the same

Comment (frederic34) [reply htdocs/comm/propal/list.php:1848]: can you find the moment where totalarray is unset using xdebug step by step ?

**Pull request closed without merging**